test/driver.js

```diff
@@ -260,13 +200,12 @@ const rasterizeAnnotationLayer = (function rasterizeAnnotationLayerClosure() {
       foreignObject.setAttribute("width", viewport.width + "px");
       foreignObject.setAttribute("height", viewport.height + "px");
       const style = document.createElement("style");
-      const stylePromise = loadStyles(styles);
       foreignObject.appendChild(style);
       const div = document.createElement("div");
       div.className = "annotationLayer";
 
       // Rendering annotation layer as HTML.
-      const [common, overrides] = await stylePromise;
+      const [common, overrides] = await Rasterize.annotationStylePromise;
```

Comment:
Please note that `static` methods can *directly* call other `static` methods and access static properties.
Hence the more compact `this.annotationStylePromise` format could be used here, and similarly elsewhere in file.